wolfcrypt/src/ecc.c

```diff
@@ -4349,17 +4349,59 @@ static int wc_ecc_shared_secret_gen_sync(ecc_key* private_key, ecc_point* point,
 #endif
 
 #ifdef WOLFSSL_HAVE_SP_ECC
+
+#if defined(WC_ECC_NONBLOCK) && defined(WC_ECC_NONBLOCK_ONLY)
+    ecc_nb_ctx_t nb_ctx;
```

Comment:
Is this at top of a brace section? Also make sure you test with `WC_ECC_NONBLOCK_ONLY` `./configure --enable-ecc=nonblock --enable-sp=yes,nonblock CFLAGS="-DWOLFSSL_PUBLIC_MP -DWC_ECC_NONBLOCK_ONLY -DECC_USER_CURVES -DHAVE_ECC384" && make`